tests/integration/rules/contains.phpt

```diff
@@ -0,0 +1,83 @@
+--FILE--
+<?php
+require 'vendor/autoload.php';
+
+use Respect\Validation\Exceptions\AllOfException;
+use Respect\Validation\Exceptions\ContainsException;
+use Respect\Validation\Validator as v;
+
+try {
+    v::contains('foo', true)->check('bar');
+} catch (ContainsException $exception) {
+    echo $exception->getMessage().PHP_EOL;
+}
+
+try {
+    v::contains('foo')->check('bar');
+} catch (ContainsException $exception) {
+    echo $exception->getMessage().PHP_EOL;
+}
+
+try {
+    v::not(v::contains('foo'))->check('fool');
+} catch (ContainsException $exception) {
+    echo $exception->getMessage().PHP_EOL;
+}
+
+try {
+    v::not(v::contains('foo', true))->check(['bar', 'foo']);
```

Comment:
The goal of the integration tests is to just test how the exceptions behave. That said we only need to test `check()`, `assert()`, `not(check())`, and `not(assert()` when they throw exception.

Could you remove some extra tests and put them to the unit test instead if they are not there already? If though it's good to have tests it's also code to maintain, and the functionality should be tested in the unit tests.

Reply:
for sure!